.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## TL;DR
```

Comment:
Overall I like it! 🚀

Reply:
Same - just a few tweaks to take it to that next level before we roll it out.